packages/core/src/lib/user.ts

```diff
@@ -29,34 +29,34 @@ export const encryptUserPassword = (
 ): {
   passwordEncryptionSalt: string;
   passwordEncrypted: string;
-  passwordEncryptionMethod: PasswordEncryptionMethod;
+  usersPasswordEncryptionMethod: UsersPasswordEncryptionMethod;
```

Comment:
key is fine to keep `passwordEncryptionMethod`

Reply:
updated